action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
   creds:

```

Comment:
The description: "Azure login wraps the az login ..." needs to be updated to specify Azure PowerShell login capability.